### DISCUSSION TITLE
What’s next? 1.10 or 2.0? 🤔

### DISCUSSION THREAD

**Discussion by DinoAwakening** [keiwando/toonsquid#380] [General]
1.10 could mean Keiwan will release an update soon, with a bit less changes. While in the other hand, 2.0; We expect a revolution out of the app. Big changes and new features, tweaks, etc… 

If I had to vote for it, Keiwan could have a whole year if he wants to, in order to provide us a MASSIVE update :D

**Comment by DWrightAnimation** [comment]
An update released today, 1.9.4, and it just has big fixes. Waiting for the next major update for ToonSquid is killing me. I'd say Christmas came early if the next major update came before Christmas, because there's so many new features that have the potential to be in the next major update.
